docs/flatcar-linux/kvm-libvirt.md

```diff
@@ -0,0 +1,295 @@
+# KVM libvirt
```

Comment:
Actually there are providers available for both Vagrant and VirtualBox, we could use some of these as well if we care about portability (as @johananl mentioned).

Reply:
Ok, good to know for later.